### ISSUE TITLE
Non-IBQ Concerns

### ISSUE THREAD

**Issue by Philip-Osterlund** [gryphonshafer/Quizzing-Rule-Book#92]
The rule-book is written, for the most part, for International quizzing.  Yet those who rely on it most are new teams and new quizzers who find many of the rules hard to follow.  The portions of the 2018 Rule Book referencing Competitive Levels have, for the most part, been left out.  Is it possible to provide a non-IBQ version?

The new rule-book does not seem any easier to read than the 2018 rule-book, but this may be due to a different ordering of material.
(NCD DQLT comments)

**Comment by jswingle** [comment]
I am of the opinion that the rulebook should, in a number of places, offer recommended alternative rules for lower levels of competition, e.g. bonuses should always be assigned seat at IBQ, but in most district-level contexts I think jumping bonuses are better. This is a large project and belongs in another issue devoted specifically to the idea.

As for the idea of providing a non-IBQ version of the current rulebook - I don't think this would be a worthwhile endeavor. There's an entire section devoted exclusively to IBQ-specific rules, under the heading "7. International Bible Quizzing" - this section is not very long, and is organized clearly and contained in this one area. I think it's much clearer and more usable than previous editions of the rulebook which had IBQ-specific rules scattered all over.

**Comment by jswingle** [comment]
On behalf of the Rules Committee:

In a vote conducted by e-mail, the Rules Committee voted 6-0 to close this issue. This means we've examined the discussion and decided that nothing ought to be changed at this time.

At the time, we don't see the need for a specific non-IBQ version of the rulebook. There is a special section of the rulebook devoted to IBQ-specific rules.

Please note that the closing of an issue doesn't mean the issue cannot be raised again in the future.

For more information on the rulebook Change Management Process, please visit: https://github.com/gryphonshafer/Quizzing-Rule-Book/blob/master/content/rule_book/change_management.md
